pkg/noobaaaccount/reconciler.go

```diff
@@ -373,6 +388,33 @@ func (r *Reconciler) CreateNooBaaAccount() error {
 		r.Logger.Errorf("got error on NooBaaAccount creation. error: %v", err)
 		return err
 	}
+	// create join secret conatining auth token for remote noobaa account
+	annotationValue, exists := util.GetAnnotationValue(r.NooBaa.GetAnnotations(), "remote-operator")
+	if exists {
+		if strings.ToLower(annotationValue) != strconv.FormatBool(false) {
+			res, err := r.NBClient.CreateAuthAPI(nb.CreateAuthParams{
+				System: r.Request.Name,
+				Role:   "operator",
+				Email:  options.OperatorAccountEmail,
+			})
+			if err != nil {
+				return fmt.Errorf("cannot create an auth token for remote operator, error: %v", err)
+			}
+			if res.Token == "" {
+				return fmt.Errorf("received empty auth token")
+			}
+			nbremoteAuthToken := &corev1.Secret{}
+			nbremoteAuthToken.Name = r.NooBaaAccount.Name
+			nbremoteAuthToken.Namespace = r.NooBaaAccount.Namespace
+			nbremoteAuthToken.StringData["auth_token"] = res.Token
+			r.Own(nbremoteAuthToken)
+
+			err = r.Client.Create(r.Ctx, r.Secret)
```

Comment:
did you mean to create `nbremoteAuthToken`?


```suggestion
			err = r.Client.Create(r.Ctx, nbremoteAuthToken)
```